src/Tizen.NUI/src/public/BaseComponents/ViewBindableProperty.cs

```diff
@@ -2675,6 +2687,27 @@ private void SetBackgroundImageBorder(Rectangle value)
             Tizen.NUI.Object.SetProperty((System.Runtime.InteropServices.HandleRef)SwigCPtr, View.Property.BACKGROUND, new PropertyValue(map));
         }
 
+        private void SetBorderlineColor(Color value)
+        {
+            if (value == null)
+            {
+                return;
+            }
+
+            (backgroundExtraData ?? (backgroundExtraData = new BackgroundExtraData())).BorderlineColor = value;
+
+            PropertyMap map = Background;
+
+            if (map.Empty())
+            {
+                return;
```

Comment:
...or, Should we ignore Selector result when backround didnt setup?
anyway, for this code now... we cannot setup borderline-only case. (Which is requested feature.)
I think we should think about this case

Reply:
I think that checking Background map can be removed because ApplyBorderline() checks Background map as follows. If it is empty, then Transparent background color is automatically set.

```
        internal virtual void ApplyBorderline()
        {
            if (backgroundExtraData == null) return;

            // ActionUpdateProperty works well only if BACKGROUND visual setup before.
            // If view don't have BACKGROUND visual, we set transparent background color in default.
            using (PropertyMap backgroundPropertyMap = new PropertyMap())
            {
                using (PropertyValue propertyValue = Object.GetProperty(SwigCPtr, Property.BACKGROUND))
                {
                    propertyValue?.Get(backgroundPropertyMap);
                    if (backgroundPropertyMap.Empty())
                    {
                        // BACKGROUND visual doesn't exist.
                        SetBackgroundColor(Color.Transparent);
                        // SetBackgroundColor function apply borderline internally.
                        // So we can just return now.
                        return;
                    }
                }
            }
```